testsuite/cluster-test/src/main.rs

```diff
@@ -853,18 +854,39 @@ impl ClusterTestRunner {
             thread::sleep(Duration::from_secs(10));
             info!("Starting...");
         }
+        let now = Utc::now();
+        let suffix = format!(
+            ".{:04}{:02}{:02}-{:02}{:02}{:02}.gz",
+            now.year(),
+            now.month(),
+            now.day(),
+            now.hour(),
+            now.minute(),
+            now.second()
+        );
+        let suffix = &suffix;
+        info!("Fill use suffix {} for log rotation", suffix);
         let jobs = self
             .cluster
             .instances()
             .iter()
             .map(|instance| {
                 let instance = instance.clone();
                 move || {
-                    if let Err(e) =
-                        instance.run_cmd_tee_err(vec!["sudo", "rm", "-rf", "/data/libra/"])
-                    {
-                        info!("Failed to wipe {}: {:?}", instance, e);
-                    }
+                    instance
+                        .run_cmd_tee_err(vec!["sudo", "rm", "-rf", "/data/libra/*db"])
+                        .map_err(|e| info!("Failed to wipe {}: {:?}", instance, e))
+                        .ok();
```

Comment:
nit: Correct me if I'm wrong, `.ok()` here and below is a no-op right? If so, can we remove it?

Reply:
`ok()` here is to suppress linter warning - otherwise it complains about unused Result